formapp/app.py

```diff
@@ -1436,9 +1440,44 @@ def endpoint_controlpanel():
             return render_template("message.html", message = resString)
 
 
-@app.route('/countdown')
+
+
+
+def timer(minutes, seconds):
+    finishedSound = "formapp/sfx_success01.wav"
+    while True:
+        print(minutes, seconds)
+        time.sleep(1)
+        seconds = seconds - 1
+        if seconds < 0:
+            minutes = minutes - 1
+            seconds = 3
+            if minutes == 0:
+                seconds = 3
+        if minutes < 0:
+            print("FINISHED")
+            playsound(finishedSound)
+            break
```

Comment:
Needs to be in the functions section of this script

This function will lock up the program while it runs
- There's no reason to use a while True with an if statement. just put the condition in the while:
- time.sleep() blocks the script from doing ANYTHING while it's sleeping, including handling GET requests.

The nonblocking part may be difficult. We need to look into asynchronous timers for Python.